src/rhub/api/scheduler/cron.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from connexion import problem
+
+from rhub.api import db
+from rhub.auth.utils import route_require_admin
+from rhub.scheduler import model
+
+
+logger = logging.getLogger(__name__)
+
+
+@route_require_admin
+def list_jobs(user):
+    cron_jobs = model.SchedulerCronJob.query.all()
+    return [i.to_dict() for i in cron_jobs]
+
+
+@route_require_admin
+def create_job(body, user):
+    cron_job = model.SchedulerCronJob.from_dict(body)
+    db.session.add(cron_job)
```

Comment:
Since `name` is `unique`, we will get a 500 internal error if we try to create the same `json` payload twice.

We could check for an existent entry `by name` (like in `#29` it's checked `by id`) and give a more friendly response and code, but this is just a nitpicking.

Reply:
Good idea. added.  I'll also add a similar check to other modules in separate MR.